turbine/src/broadcast_stage/broadcast_utils.rs

```diff
@@ -96,6 +99,34 @@ pub(super) fn recv_slot_entries(receiver: &Receiver<WorkingBankEntry>) -> Result
     })
 }
 
+// Returns the Merkle root of the last erasure batch of the parent slot.
+pub(super) fn get_chained_merkle_root(
```

Comment:
this function might be more intuitive if it was named `get_chained_merkle_root_from_parent`

Reply:
renamed